starboard/nplb/posix_compliance/posix_string_compare_no_case_n_test.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "starboard/common/string.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+namespace starboard {
+namespace nplb {
+namespace {
+
+#if SB_API_VERSION >= 16
+TEST(SbStringCompareNoCaseNTest, SunnyDaySelf) {
```

Comment:
Please rename the tests to avoid confusion with the exiting NPLB tests.

Reply:
Lol, you're right. Changed it. Definitely not at peak performance today